src/doc/DocAnnotator.js

```diff
@@ -532,11 +537,18 @@ class DocAnnotator extends Annotator {
             this.selectionEndTimeout = null;
         }
 
+        const isClickOutsideCreateDialog = this.isCreatingHighlight && util.isInDialog(event);
+        if (isClickOutsideCreateDialog) {
+            this.hideCreateDialog(event);
```

Comment:
do we need to set `this.isCreatingHighlight` to false as well?

Reply:
yes. I'll add that

Reply:
That is actually done in `hideCreateDialog()`